### ISSUE TITLE
Problem using "\" in a Webacceleration profile.

### ISSUE THREAD

**Issue by antec0042** [F5Networks/f5-appsvcs-extension#691]
### Environment
 * Application Services Version: 3.42.0
 * BIG-IP Version: 15.1.8

### Summary
When posting something similar to this:

        "my-webacceleration": {
          "class": "HTTP_Acceleration_Profile",
          "parentProfile": {
                    "bigip": "/Common/webacceleration"
          },
          "agingRate": 9,
          "ignoreHeaders": "all",
          "insertAgeHeaderEnabled": true,
          "maximumAge": 300,
          "maximumEntries": 10000,
          "maximumObjectSize": 5000000,
          "minimumObjectSize": 10,
          "cacheSize": 200,
          "uriIncludeList": [
            "*.gif",
            "*.svg",
            "/XXX/XXX/rest/test/hello/prefix\\?key=ms\\.spa\\."
          ]
        }

The result string in the include list should be:

"/XXX/XXX/rest/test/hej/prefix\\?key=ms\\.spa\\."

However, all the backslashes gets removed. No matter how many used. I've tried up to 5.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "schemaVersion": "3.14.0",
    "class": "ADC",
    "Tenant": {
        "class": "Tenant",
        "Application": {
            "template": "generic",
            "vip": {
                "class": "Service_HTTP",
                "pool": "web_pool",
                "virtualAddresses": ["192.0.2.0"]
            },
        "my-webacceleration": {
          "class": "HTTP_Acceleration_Profile",
          "parentProfile": {
                    "bigip": "/Common/webacceleration"
          },
          "agingRate": 9,
          "ignoreHeaders": "all",
          "insertAgeHeaderEnabled": true,
          "maximumAge": 300,
          "maximumEntries": 10000,
          "maximumObjectSize": 5000000,
          "minimumObjectSize": 10,
          "cacheSize": 200,
          "uriIncludeList": [
            "*.gif",
            "*.svg",
            "/XXX/XXX/rest/test/hello/prefix\\?key=ms\\.spa\\."
          ]
        }
        }
    }
}
```

2. Check the newly created my-webacceleration profile on the big-ip:

The result string is: "/XXX/XXX/rest/test/hej/prefix?key=ms.spa."

### Expected Behavior

The wanted string should be: "/XXX/XXX/rest/test/hej/prefix\\?key=ms\\.spa\\."

**Comment by antec0042** [comment]
I see my backslashes got removed also in this tool :)

Result string should be:
"/XXX/XXX/rest/test/hej/prefix\\?key=ms\\.spa\\."

**Comment by sunitharonan** [comment]
Hi @antec0042, thank you for reaching out to us. Please contact us at  automation_toolchain_pm@f5.com so we can prioritize this issue.

**Comment by RobCupples** [comment]
@sunitharonan This was resolved in 3.44.0 under this release note:

`HTTP_Acceleration_Profile does not handle backslashes in uriExcludeList, uriIncludeList, uriIncludeOverrideList, and uriPinnedList properties, https://github.com/F5Networks/f5-appsvcs-extension/issues/691`
